controllers/nc_news_controllers.js

```diff
@@ -40,7 +41,8 @@ const getArticles = (req, res, next) => {
 
 const getCommentsByArticleID = (req, res, next) => {
     const { article_id } = req.params
-    Promise.all([selectArticleByID(article_id), selectCommentsByArticleID(article_id)]).then((promiseArr) => {
+    Promise.all([selectArticleByID(article_id), selectCommentsByArticleID(article_id)])
+    .then((promiseArr) => {
```

Comment:
I would ideally call the variable in the .then something more descriptive, such as comments